### ISSUE TITLE
"The 'chance' to run a command when killing a [Mob] is not a number."

### ISSUE THREAD

**Issue by Weasel-Beans** [Rocologo/MobHunting#540]
I've been getting this issue:

>.... [01:40:33 INFO]: [MobHunting] The 'chance' to run a command when killing a Zombie Pigman is not a number.
>.... [01:40:35 INFO]: [MobHunting] The 'chance' to run a command when killing a Zombie Pigman is not a number.
>.... [01:40:40 INFO]: [MobHunting] The 'chance' to run a command when killing a Zombie Pigman is not a number.
>.... [01:40:42 INFO]: [MobHunting] The 'chance' to run a command when killing a Zombie Pigman is not a number.

With every mob it seems.

Here's my config: https://pastebin.com/UGEzTutX

**Comment by Rocologo** [comment]
It looks like a number to me... I will have to check if there is a problem with small numbers.

**Comment by Rocologo** [comment]
I found the bug. The chance must be a string. Check this 
```
    - chance: '0.2'
      message_type: ActionBar
      cmd: give {player} Iron_ingot 1
      message: You got an iron ingot!
```

**Comment by Rocologo** [comment]
Did you get it work?

**Comment by Weasel-Beans** [comment]
Ah I see, it looks like I messed up the ' ' symbols around the chances, as well as the money reward for a few things such as ender dragon and wither. I changed it to follow exactly how it's done in the example and it seems to have fixed it.

I think I probably made that mistake months ago, glad I finally figured it out. Thank you. This issue is essentially the same as the other one I opened I believe.
